packages/g/gdbuspp/package.yml

```diff
@@ -0,0 +1,18 @@
+name       : gdbuspp
+version    : 3
```

Comment:
The only thing I'm not sure of is if this needs to be quoted so YAML knows it's a string. I can never remember when we do and don't have to do that.

Reply:
Definitely needs to be quoted, good catch!